frontend/src/app/main/data/common.cljs

```diff
@@ -155,3 +155,41 @@
                          :files files
                          :binary? binary?}))))))))
 
+;;;;;;;;;;;;;;;;;;;;;;
+;; File info
+;;;;;;;;;;;;;;;;;;;;;;
+
+(defn get-file-info
+  [on-info params]
+  (ptk/reify ::get-file-info
+    ptk/WatchEvent
+    (watch [_ _ _]
+      (->> (rp/cmd! :get-file-info params)
+           (rx/map on-info)))))
+
+;;;;;;;;;;;;;;;;;;;;;;
+;; Team info
+;;;;;;;;;;;;;;;;;;;;;;
+
+(defn get-team-info
```

Comment:
You don't need to define events if you just want to make a request and call the callback, perform the request directlly

Reply:
fixed